contrib/pg_tde/documentation/docs/variables.md

```diff
@@ -0,0 +1,47 @@
+# GUC Variables
+
+The `pg_tde` extension provides GUC variables to configure the behaviour of the extension:
+
+## pg_tde.wal_encrypt
+
+A `boolean` variable controlling if WAL writes are encrypted or not.
+
+Changing this variable requires a server restart, and can only be set at the server level.
+
+WAL encryption is controlled globally, if enabled all WAL writes are encrypted.
+
+This variable only controls new writes to the WAL, it doens't affect existing WAL records.
+`pg_tde` is always capable of reading existing encrypted WAL records, as long as the keys used for the encryption are still available.
+
+
+
+Enabling WAL encryption requires a configured global principal key.
+
+The default value is false.
+
+## pg_tde.enforce_encryption
+
+A `boolean` variable controlling if creation of new not encrypted tables is enabled or not.
+
+This can be set at global, database, user or session level, but changing the value requires superuser permissions.
```

Comment:
What does the user level mean?

Reply:
Postgres allow GUC variables on multiple levels:
* Global
* Per database (set for specific databases)
* Per user (set for specific users)
* Per session (set for the current session)